pandas/core/resample.py

```diff
@@ -498,23 +498,142 @@ def pad(self, limit=None):
 
         See Also
         --------
-        Series.fillna
-        DataFrame.fillna
+        backfill: Backward fill the new missing values in the resampled data.
+        fillna : Fill ``NaN`` values using the specified method, which can be
+            'backfill'.
+        pad : Forward fill ``NaN`` values.
+        pandas.Series.fillna : Fill ``NaN`` values in the Series using the
+            specified method, which can be 'backfill'.
+        pandas.DataFrame.fillna : Fill ``NaN`` values in the DataFrame using
+            the specified method, which can be 'backfill'.
+
+        Examples
+        --------
+
+        Resampling a Series:
+
+        >>> s = pd.Series([1, 2, 3],
+        ...               index=pd.date_range('20180101', periods=3,
+        ...                                   freq='1h'))
+        >>> s
+        2018-01-01 00:00:00    1
+        2018-01-01 01:00:00    2
+        2018-01-01 02:00:00    3
+        Freq: H, dtype: int64
+
+        >>> s.resample('20min').nearest()
+        2018-01-01 00:00:00    1
+        2018-01-01 00:20:00    1
+        2018-01-01 00:40:00    2
+        2018-01-01 01:00:00    2
+        2018-01-01 01:20:00    2
+        2018-01-01 01:40:00    3
+        2018-01-01 02:00:00    3
+        Freq: 20T, dtype: int64
+
+        Resample in the middle:
+
+        >>> s.resample('30min').nearest()
+        2018-01-01 00:00:00    1
+        2018-01-01 00:30:00    2
+        2018-01-01 01:00:00    2
+        2018-01-01 01:30:00    3
+        2018-01-01 02:00:00    3
+        Freq: 30T, dtype: int64
+
+        Limited fill:
+
+        >>> s.resample('10min').nearest(limit=1)
+        2018-01-01 00:00:00    1.0
+        2018-01-01 00:10:00    1.0
+        2018-01-01 00:20:00    NaN
+        2018-01-01 00:30:00    NaN
+        2018-01-01 00:40:00    NaN
+        2018-01-01 00:50:00    2.0
+        2018-01-01 01:00:00    2.0
+        2018-01-01 01:10:00    2.0
+        2018-01-01 01:20:00    NaN
+        2018-01-01 01:30:00    NaN
+        2018-01-01 01:40:00    NaN
+        2018-01-01 01:50:00    3.0
+        2018-01-01 02:00:00    3.0
+        Freq: 10T, dtype: float64
+
+        Resampling a DataFrame that has missing values:
+
+        >>> df = pd.DataFrame({'a': [2, np.nan, 6], 'b': [1, 3, 5]},
+        ...                   index=pd.date_range('20180101', periods=3,
+        ...                                       freq='h'))
+        >>> df
+                               a  b
+        2018-01-01 00:00:00  2.0  1
+        2018-01-01 01:00:00  NaN  3
+        2018-01-01 02:00:00  6.0  5
+
+        >>> df.resample('20min').nearest()
+                               a  b
+        2018-01-01 00:00:00  2.0  1
+        2018-01-01 00:20:00  2.0  1
+        2018-01-01 00:40:00  NaN  3
+        2018-01-01 01:00:00  NaN  3
+        2018-01-01 01:20:00  NaN  3
+        2018-01-01 01:40:00  6.0  5
+        2018-01-01 02:00:00  6.0  5
+
+        Resampling a DataFrame with shuffled indexes:
```

Comment:
I am not fully sure this example is needed, as it is something general to `resample`, and not specific to the  `nearest` method